### ISSUE TITLE
Is it possible to implement this via Group Policy?

### ISSUE THREAD

**Issue by umarbutler** [Upinel/BetterRDP#5]
Thanks for creating this. I'd like to know whether any or all of these settings can be enabled via Group Policy instead of Registry?

**Comment by stephenc01** [comment]
Yes. Any reg key can be deployed via gpo. The difference is that it will be automatically undone when you remove the gpo. From memory there is a section to import it.

**Comment by GloriaOS** [comment]
Great suggestion - and yes, absolutely! I've added a full GROUP_POLICY.md guide covering both Group Policy Preferences (GPP Registry Extension) and Startup Script deployment. Complete key-value table ready to drop into GPMC. Thanks for the nudge!
